R/bs.R

```diff
@@ -101,20 +101,41 @@ step_bs_new <-
   }
 
 bs_wrapper <- function(x, args) {
```

Comment:
Please call this `bs_statistics()` since it just does those computations.

Reply:
Done.